src/Workspaces/Core/Portable/Storage/SQLite/v2/SQLitePersistentStorageService.cs

```diff
@@ -15,24 +16,23 @@
 
 namespace Microsoft.CodeAnalysis.SQLite.v2
 {
+    /// <summary>
+    /// The storage service is a process wide singleton.  It will ensure that any workspace that
+    /// wants to read/write data for a particular solution gets the same DB.  This is important,
+    /// we do not partition access to the information about a solution to particular workspaces.
```

Comment:
❗ This was already managed through `SQLiteConnectionPoolService`.

Reply:
The connection pool service ensures only one writer to any db in the process at a time. It does not accomplish what this pr accomplishes though. Specifically, multiple workspaces in the process using the same db.  Today, each workspace will get a separate db, which is not desirable.